SPLASH/src/game/states/MenuState.cpp

```diff
@@ -716,6 +749,94 @@ void MenuState::renderOptions() {
 		m_optionsWindow.renderWindow();
 	}
 	ImGui::End();
+}
+void MenuState::renderCredits(float dt) {
+
+	ImGui::SetNextWindowPos(m_pos);
+	ImGui::SetNextWindowSize(m_size);
+	ImGui::SetNextWindowSizeConstraints(m_minSize, m_maxSize);
+
+	struct Info {
+		std::string name;
+		std::string info;
+	};
+	static std::map<std::string, Info> names;
+	names["Johansson"] = { "Henrik", "person of options" };
+	names["Junede"] = { "Fredrik", "person of artificial cleaning" };
+	names["Wester"] = { "Alexander", "person of pew" };
+	names["Bj�rk"] = { "Gustav", "person of scuffed newton" };
+	names["Wahl"] = { "Emil", "person of slave driving" };
+	names["Fredriksson"] = { "Daniel", "person of Github love" };
+	names["Glandberger"] = { "Oliver", "person of curry (with salad)" };
+	names["Fasth"] = { "Tobias", "person of wet balls" };
+	names["Enfeldt"] = { "Viktor", "person of Synchronization" };
+	names["Asp"] = { "Samuel", "person of pain and suffering" };
+	names["Meunier"] = { "Peter", "person of clutter" };
+	names["Bengtsson"] = {"David", "person of Jet engines" };
```

Comment:
Does not really matter, but be aware that this code will make a static map only once, but still update the names every function call.

Reply:
oh right, didnt think about that